godtools/Managers/LanguagesManager.swift

```diff
@@ -19,13 +19,29 @@ class LanguagesManager: GTDataManager {
     let path = "/languages"
     
     var languages = [Language]()
+    var selectingPrimaryLanguage = true
     
     override init() {
         super.init()
         serializer.registerResource(LanguageResource.self)
     }
     
-    func loadFromDisk() -> Promise<[Language]> {
+    func loadFromDisk(id: String) -> Language {
+        let language = Language.mr_findFirst(byAttribute: "remoteId", withValue: id)!
```

Comment:
As I said before, here you can call a method to search by remoteId instead of calling the method of MR, but the difference is that here you don't set the context, and previously you do, is there a reason?